kinetic/distribution.yaml

```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 141: http://ros.org/reps/rep-0141.html
+---
+release_platforms:
```

Comment:
I removed them because I wasn't sure we were building packages for Fedora in the buildfarm. Fedora isn't listed in https://github.com/ros-infrastructure/rep/pull/106, should I update that too?